content/bitcoin-core.md

```diff
@@ -40,11 +40,7 @@ Similarly, the `getrawaddrman` RPC command's response will include `source_mappe
 
 #### Inspecting assignment changes between two ASmaps
 
-The `contrib/asmap-tool.py` script provides a `diff-addrs` command to show changes in AS mappings between two ASmaps. The output will show the peer IP addresses, and their new status: assigned (or reassigned) from previous AS to the new AS. It takes a set of peers in the format of the `getnodeaddresses` RPC, and two different ASmaps to measure changes as arguments:
-```
-$ contrib/asmap/asmap-tool.py diff-addrs <asmap-1> <asmap-2> <(bitcoin-cli getnodeaddresses 0)
-```
-This allows a node operator to see the changes that a known IP address's AS assignment underwent in a given timespan, defined by the times at which the ASmaps was generated. If an `unassigned` value is returned as either the original or latest AS assignment, it means that the ASmap provided did not have a mapping for the IP address given.
+The `contrib/asmap-tool.py` script provides the `diff` and `diff-addrs` command to show changes in AS mappings between two ASmaps. See the Core [documentation](https://github.com/bitcoin/bitcoin/tree/master/contrib/asmap) for usage.
```

Comment:
Not sure if every reader understand what is meant by just "Core" and I would like to the README file directly.

```suggestion
The `contrib/asmap-tool.py` script provides the `diff` and `diff-addrs` command to show changes in AS mappings between two ASmaps. See the Bitcoin Core [documentation](https://github.com/bitcoin/bitcoin/tree/master/contrib/asmap/README.md) for usage.
```